modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= BYO VPC with multiple availability zones required fields
+
+. Create six subnets as follows:
```

Comment:
There should be notification show that the subnets should be in 3 different zones. Each zone has 1 private subnet and 1 public subnet.